hacks/run-tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+CYPRESS_BASE_URL=${CYPRESS_BASE_URL:-http://192.168.0.126:3000} # URL of running Metal3 Installer UI
+TEST_IMAGE=${TEST_IMAGE:-"quay.io/mareklibra/facet-tests:123"}
+BASE_DIR=${BASE_DIR:-"`pwd`"/`date +%D_%T|sed 's/\//_/g'|sed 's/:/-/g'`}
+
+echo Connecting to UI at: ${CYPRESS_BASE_URL}
+echo Test image: ${TEST_IMAGE}
+
+VIDEO_DIR=${BASE_DIR}/videos
+SCREENSHOT_DIR=${BASE_DIR}/screenshots
+
+mkdir -p ${VIDEO_DIR}
+mkdir -p ${SCREENSHOT_DIR}
+docker run -it \
```

Comment:
We should use either docker or podman across our scripts (Makefile uses podman).

Reply:
done